Emby.Server.Implementations/Playlists/PlaylistManager.cs

```diff
@@ -204,7 +226,14 @@ private void AddToPlaylistInternal(string playlistId, IEnumerable<Guid> itemIds,
 
             foreach (var item in items)
```

Comment:
I think it should be possible to filter out the duplicates in a much simpler way, something like this:
```
var existingIds = playlist.LinkedChildren.Select(c => Guid.Parse(c.Id))
var uniqueIds = itemIds.Distinct().Except(existingIds);
```
This is much more readable to me, and should also be faster than checking each id in a loop. The call to `Distinct()` also handles the case where `itemIds` contains duplicate entries